addons/im_livechat/static/src/components/discuss/tests/discuss_tests.js

```diff
@@ -378,6 +378,68 @@ QUnit.test('call buttons should not be present on livechat', async function (ass
     );
 });
 
+QUnit.test('reaction button should not be present on livechat', async function (assert) {
+    assert.expect(1);
+
+    this.data['mail.channel'].records.push({
+        channel_type: 'livechat',
+        id: 10,
+        livechat_operator_id: this.data.currentPartnerId,
+        members: [this.data.currentPartnerId, this.data.publicPartnerId],
+    });
+    await this.start({
+        discuss: {
+            params: {
+                default_active_id: 'mail.channel_10',
+            },
+        },
+    });
+    await afterNextRender(() => {
+        document.querySelector(`.o_ComposerTextInput_textarea`).focus();
+        document.execCommand('insertText', false, "Test");
+    });
+    await afterNextRender(() =>
+        document.querySelector('.o_Composer_buttonSend').click()
+    );
+    await afterNextRender(() => document.querySelector('.o_Message').click());
+    assert.containsNone(
+        document.body,
+        '.o_MessageActionList_actionReaction',
+        "should not have action to add a reaction"
+    );
```

Comment:
In forward-port:
- use `pyEnv` instead of `this.data`.
- use `click` and `insertText` returned by `this.start` (instead of `afterNextRender(() => { .click()/insertText() })`.